addons/knowledge/static/src/scss/knowledge_frontend_views.scss

```diff
@@ -10,6 +10,12 @@
             }
         }
     }
+    .o_knowledge_article:not(.o_full_width) {
+        width: 750px;
```

Comment:
@dbeguin I can not answer the previous questions :smiling_face_with_tear: Github shows me an error (can not resolve id....)

> what if you use a super wide screen ?

The editor will take all available space. Should I add a max-width constraint ?

> why a different width than in backend ?

When I used the same width, I noticed that the line-break was different: 750px in the frontend was not equivalent then 750px in the backend. I think it has something to do with the viewport meta-tag that defines a different initial zoom level. To have something consistent, I would use the `rem` unit. Something like 40rem ?